__tests__/project.test.js

```diff
@@ -166,3 +168,38 @@ describe("/api/articles/:article_id/comments", () => {
       });
   });
 });
+
+describe("/api/articles/:article_id/comments", () => {
+  test("accepts a body with username and body responds with the posted comment", () => {
+    return request(app)
+      .post("/api/articles/2/comments")
+      .send({ username: "butter_bridge", body: "I really like Wing Chun" })
+      .expect(201)
+      .then(({ body }) => {
+        expect(body.article).toHaveProperty("comment_id", 19);
+        expect(body.article).toHaveProperty("body", "I really like Wing Chun");
+        expect(body.article).toHaveProperty("article_id", 2);
+        expect(body.article).toHaveProperty("author", "butter_bridge");
+        expect(body.article).toHaveProperty("votes", 0);
+        expect(body.article).toHaveProperty("created_at");
+      });
+  });
+  test("returns a message if username doensn't exist", () => {
+    return request(app)
+      .post("/api/articles/2/comments")
+      .send({ username: "bruce_lee", body: "I really like Wing Chun" })
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Invalid input");
+      });
+  });
+  test("returns a message if the article doesn't exist", () => {
+    return request(app)
+      .post("/api/articles/999/comments")
+      .send({ username: "butter_bridge", body: "I really like Wing Chun" })
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Invalid input");
+      });
+  });
```

Comment:
Nice test as well, but dont forget to also test for when we are given an invalid article_id. I.e. /api/articles/banana/comments